content/en/load-testing/Manual/060-test-development.md

```diff
@@ -60,9 +60,9 @@ This is just an example and a different breakdown of flows and actions might mak
 
 ### Local Testing and Test Consolidation
 
-XLT tests are basically JUnit tests, so they can be run (and debugged) from the IDE, and you should do so repeatedly for consolidating the tests you wrote, because you really want stable and predictable tests for your load test. 
+XLT tests are basically JUnit tests; they can be run (and debugged) from the IDE. This should be done repeatedly while consolidating the various tests you have writen until a high level of stability is reached. Load testing necessitates stable and predictable test runs. Otherwise you will keep running into funtional test error and this can ultimately give you a false picture of your system's load capabilities.
```

Comment:
have writ**t**en (sorry I missed this one before)

Reply:
Fixed.